node/src/run_modes_factories.rs

```diff
@@ -192,37 +191,65 @@ mod tests {
 
     #[test]
     fn make_for_daemon_initializer_factory_labours_hard_and_produces_a_proper_object() {
+        use std::ptr::addr_of;
         let daemon_clustered_params = test_clustered_params();
+        let init_pointer_of_recipients_factory =
+            addr_of!(*daemon_clustered_params.recipients_factory);
+        let init_pointer_of_channel_factory = addr_of!(*daemon_clustered_params.channel_factory);
+        let init_pointer_of_rerunner = addr_of!(*daemon_clustered_params.rerunner);
         let subject = DaemonInitializerFactoryReal::new(
             Box::new(NodeConfiguratorInitializationReal),
             daemon_clustered_params,
         );
-        let port = find_free_port();
-        let args = convert_str_vec_slice_into_vec_of_strings(&[
+        let args = &[
             "program",
             "--initialization",
             "--ui-port",
-            &port.to_string(),
-        ]);
+            1234.to_string().as_str(),
+        ]
+        .array_of_borrows_to_vec();
         let mut stream_holder = FakeStreamHolder::default();
         let result = subject.make(&args, &mut stream_holder.streams()).unwrap();
```

Comment:
Please convert this to standard form, with blank lines before and after the call to the method under test.